src/core/pointcloud/qgspointcloudstatistics.h

```diff
@@ -77,6 +77,9 @@ class CORE_EXPORT QgsPointCloudStatistics
     //! Constructor
     QgsPointCloudStatistics();
 
+    //! Constructor from JSON object
+    QgsPointCloudStatistics( const QJsonObject &obj );
```

Comment:
My preference for the API to read/write JSON would be:
- read: `static QgsPointCloudStatistics fromCopcStatistics( const QString &data, QString &errorMessage );`
- write: `QString toCopcStatistics() const;`

Comments:
- better to use QString to read/write - it's more general than QJsonObject, and maybe if the format is changed at some point, we're not stuck with QJsonObject in the API
- better to use static read function rather than a constructor
- let's have basic error reporting when stats reading fails for some reason

Reply:
I think it is better if we put JSON in the name of read/write function so that nobody gets confused about it.
And for the future if we decide to change to another format we should use a different EVLR ID and keep the code for JSON reading and writing since some generated indexed datasets might contain it.
Also the conversion of statistics to a string in any format is quite unrelated to COPC strictly speaking and we could end up wanting to save statistics of an EPT dataset for example in the project if we wanted that and that could be  especially convenient for remote dataset.